src/Widgets/SettingsPopover.vala

```diff
@@ -20,11 +20,9 @@ public sealed class Terminal.SettingsPopover : Gtk.Popover {
     }
 
     private const string STYLE_CSS = """
-        .color-button radio {
+        .color-button.%s radio {
             background-color: %s;
             color: %s;
-            padding: 10px;
```

Comment:
When I did the same in the Gtk4 branch, the buttons became small so is this necessary here?

Reply:
I just moved this to the application.css since it never changes

Reply:
Oh I see - I missed that.